x-pack/test/functional/fixtures/kbn_archiver/spaces/enter_space.json

```diff
@@ -0,0 +1,33 @@
+{
+  "attributes": {
+    "buildNum": 8467,
+    "dateFormat:tz": "UTC",
+    "defaultRoute": "http://example.com/evil"
+  },
+  "coreMigrationVersion": "8.4.0",
+  "id": "6.0.0",
```

Comment:
__issue:__ hmm, this will just create a second `config` object for Kibana `6.0.0` that won't be used. We need to this id to match the version of the Kibana instance.

It looks like `kibanaServer.importExport.load` doesn't allow you to dynamically change the id, and we'd need to use `kibanaServer.savedObjects` instead. Do you think something like this would be possible?
```ts
// Assets that config object really exists
const config = await kibanaServer.savedObjects.get({
  id: await kibanaServer.version.get(),
  type: 'config',
});

// Update config object
await kibanaServer.savedObjects.update({
  id: config.id,
  type: config.type,
  attributes: { defaultRoute: 'http://example.com/evil' },
});
```

Reply:
Shouldn't the tests fail then?